controllers/operatorconfigmap_controller.go

```diff
@@ -123,23 +128,54 @@ func (c *ClusterVersionReconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.8.3/pkg/reconcile
-func (c *ClusterVersionReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+func (c *OperatorConfigMapReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	var err error
 	c.ctx = ctx
-	c.log = log.FromContext(ctx, "ClusterVersion", req)
-	c.log.Info("Reconciling ClusterVersion")
+	c.log = log.FromContext(ctx, "OperatorConfigMap", req)
```

Comment:
recently I observed that name of reconciler and req is already part of `ctx` and this line is adding them to the log again, could you pls check and change this to only `c.log = log.FromContext(ctx)`

Reply:
Yes @leelavg,  you are right.
The context passed to the reconcile function (from the controller) already contain the request object